.gitignore

```diff
@@ -8,4 +8,5 @@ project/plugins/project/
 .cache
 .lib/
 .idea/
-.idea_modules/
\ No newline at end of file
+.idea_modules/
+.sc
```

Comment:
What's this?

Reply:
IntelliJ Scala Worksheet. I often try out stuff, and then by accident commit them.